src/github/client/interceptors.ts

```diff
@@ -18,6 +19,17 @@ export const setRequestStartTime = (config) => {
 	return config;
 };
 
+/**
+ * Sets the timeout to the request based on the github-client-timeout feature flag
+ */
+export const setRequestTimeout = async (config: AxiosRequestConfig): Promise<AxiosRequestConfig> => {
+	const timeout = await numberFlag(NumberFlags.GITHUB_CLIENT_TIMEOUT, 30000);
+	if(timeout) {
```

Comment:
This will always return true?

Reply:
unless you return `0` as a number flag, which I really doubt would happen

Reply:
Nope, timeout can be 0

Reply:
Wouldn't it just return the defaultValue when it's off?

Reply:
You're also not checking if the config has a value already that's been explicitly set.  I'd probably go with the approach of `config.timeout = config.timeout || await numberFlag(NumberFlags.GITHUB_CLIENT_TIMEOUT, 60000);`

Reply:
No the defaultValue will be returned when LaunchDarkly is offline. Otherwise LD will return the value which we set in LD when targeting is OFF. 

>You're also not checking if the config has a value already that's been explicitly set.
We don't set it explicitly in requests so far. Doesn't make sense to add this check now. We'll get rid of the interceptor as soon as we clean up the flag.